src/frontend/Komponenter/Behandling/VedtakOgBeregning/Overgangsstønad/InnvilgeVedtak/InntektsperiodeValg.tsx

```diff
@@ -91,153 +114,286 @@ const InntektsperiodeValg: React.FC<Props> = ({
         ]);
     };
 
+    const oppdaterAvhukningsvalg = (inntektstyper: EInntektstype[]) => {
+        const avhukerKolonneMedVerdi = (
```

Comment:
Ville lagt `avhukerKolonneMedVerdi` og i hvertfall `settFeilmeldingAvhukingMedType` i utils filen i samme mappe. Da blir det lettere å se hva denne funksjonen faktisk gjør. Den burde heller ikke ha ansvar for å opprette disse to metodene selv.

Reply:
Ville splittet opp `avhukerKolonneMedVerdi` i to konstanter. En som gjør de to første sjekkene og heter `checkBoxAvkrysset` og en som gjør den siste sjekken og heter `inntektsperioderHarVerdiForKolonne`. Deretter ville jeg laget en const som heter `skalViseFeilmelding` som sjekker på de to nye nevnte konstantene. 

Det blir kanskje en linje mer med kode eller to, men koden blir også mer forståelig.

Reply:
✅